reader-sdk-react-native-quickstart/README.md

```diff
@@ -66,8 +66,8 @@ problems building the sample app.
    [root README]for this repo:
     ```bash
     ruby <(curl https://connect.squareup.com/readersdk-installer) install \
-    --app-id YOUR_SQUARE_READER_APP_ID                                    \
-    --repo-password YOUR_SQUARE_READER_REPOSITORY_PASSWORD
+    --app-id $YOUR_SQUARE_READER_APP_ID                                    \
```

Comment:
Thanks for your contribution! Can you also update the instructions a few lines above to reflect the changes? So, ```Install ReaderSDK, replacing `YOUR_SQUARE_READER_APP_ID` and
   `YOUR_SQUARE_READER_REPOSITORY_PASSWORD` with your Reader SDK credentials.``` should probably be something like ```...replacing `$YOUR_SQUARE_READER_APP_ID`... or define variable YOUR_SQUARE_READER_APP_ID```.